docs/Makefile

```diff
@@ -13,8 +13,12 @@ clean:
 	rm -rf $(CURDIR)/website/public
 	rm -rf $(CURDIR)/website/resources
 
+.PHONY: generate-cli-docs
```

Comment:
If we don't commit `cli.md`, and we only generate the like this, we'll never have CLI docs for previous versions, only for the latest one.

Reply:
Fair. When do you suggest we commit this? I guess it would need to be done in the CI pipeline, right? Any previous examples of us doing that there?

Reply:
We could try to generate it on the fly for all checked out versions... But that's cumbersome since we only check out markdown content right now and would need all code for that.

Check the wasm updater, it does something similar in a GitHub workflow.

Reply:
https://github.com/open-policy-agent/opa/blob/main/.github/workflows/post-merge.yaml#L9